### DISCUSSION TITLE
keyball44 ランドを剥がしてしまいました。

### DISCUSSION THREAD

**Discussion by Yusei-Ichikawa** [Yowkees/keyball#262] [Q&A]
![IMG_9632](https://github.com/Yowkees/keyball/assets/115162349/e418a344-7bb1-4184-be96-8103cd8d369c)

お忙しい中失礼します。
トラックボール有側のLEDのランドを2つ剥がしてしまいました。
配線はどこに繋げば良いでしょうか？
ランドが剥がれた場所は写真の10番、左上と左下です。
どうかご教授お願い致します。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
１０番のLEDは、まだ生きているパッド２つに半田付けして固定していただいた後に、
下記画像の同色どうしをエナメル線等で接続してください。これで復活できると思います。
お手数ですがご対応お願いいたします。

![44R_LEDfix](https://github.com/Yowkees/keyball/assets/84209772/92986128-83cb-45fa-b531-2b146c72fd4f)

**Reply by Yusei-Ichikawa** [reply]
![image](https://github.com/Yowkees/keyball/assets/115162349/a42808a1-b34d-42d5-aa0a-f371b9768aaa)
お返事ありがとうございます。

引き続き同じ箇所の質問をさせて下さい。

同じく10番の右側2つの配線の繋げる先を教えて頂きたいです。

導線で繋いでいる際に上向きの力を入れてしまい、右側のランドも剥がれてしまった可能性があります。

導線で繋ぐ前にLED自体を新しいものに変えており、その後テスト運転をしたのですがやはり10番から点灯していません。

どうかご教授をよろしくお願い致します。

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありません。
下記画像の同色２本をつなげていただければ復活できると思います。緑色は距離が遠いのでご注意ください。
![44RLEDfix](https://github.com/Yowkees/keyball/assets/84209772/ca23869b-12f9-4a78-80fc-b9d432d464b1)

**Reply by Yusei-Ichikawa** [reply]
ご返信ありがとうございます！
無事にLEDは全て光りました！

すみませんまた新たに問題が出たのでご相談させて下さい。

![image](https://github.com/Yowkees/keyball/assets/115162349/0f626f92-900a-471d-85b9-15c753eeb1c3)

![image](https://github.com/Yowkees/keyball/assets/115162349/e4ac18b1-30c3-4ef7-a498-43d0b73615bf)

トラックボール無し側のキーに反応が無いものが2つあります。
写真1枚目のキーキャップが着いていない端側2つのキーです。
写真2枚目はその裏側ですが、目視では分かりにくいので原因の可能性がある場所を教えて頂きたいです！

**Comment by Yowkees** [comment]
うーん。その２キーだけとなると関連性がない２キーですので、キーソケットの半田付けがあやしいと思います。
ピンセットでキーソケット両端をショートするとどうなるかなどで問題を切り分けて調査いただければ幸いです。

**Reply by Yusei-Ichikawa** [reply]
度々すみません。

キーソケットをピンセットでショート等しましたが、同じ位置のCherry MX、lowProfile両方のキーソケットが反応しません。

表裏両方ピンセットでショートしましたがダメでした。

ランドに着いている半田にピンセットを当てても反応しないという事は、基盤の他の所に問題があるのでしょうか？

**Reply by Yusei-Ichikawa** [reply]
今気づいたのですが、もしかしたらここのダイオードの向きが逆ですかね？

そうしたらこの2つのスイッチの不良と関係がありますか？

![image](https://github.com/Yowkees/keyball/assets/115162349/35cb688f-88d8-4614-8421-d2f33b211dec)

**Reply by Yusei-Ichikawa** [reply]
写真のダイオードの向きを変えたらできました！

これでもう問題は全て解決できました！

本当に拙い質問に答えて下さりありがとうございました🙇🏻‍♂️

keyballを使い倒す日々が楽しみです！！

**Comment by Yowkees** [comment]
ダイオードの向きが逆でしたか。ご自身で解決いただいてありがとうございます。
ぜひ末永くKeyballをご愛用いただければ幸いです。
今後ともどうぞよろしくお願いいたします。
